FNPlugin/Collectors/SolarWindCollector.cs

```diff
@@ -141,41 +156,63 @@ public override void OnFixedUpdate()
             }
         }
 
```

Comment:
Do you mean the UpdateDistanceInGUI function or the atmosphere check above it? Sorry, either github is not showing where this comment should be clearly or I'm too tired to see clearly (which is true x-x).
If it's about the distance - after changing the concentration function to depend on solarFlux as you recommended, I thought about taking the distance out altogether. In the end I decided to keep it around (and show it in the part's GUI) because it quite clearly communicates to the player that the distance plays a role in the concentration, and that they should move closer if they want to get better results.